src/main/python/ccrf/prepare_test_data.py

```diff
@@ -10,61 +10,69 @@
 
 from utils import TfidfTgzReader
 
-topic_list = [
-    '321', '336', '341',
-    '347', '350', '362',
-    '363', '367', '375', '378', '393',
-    '397', '400', '408', '414',
-    '422', '426', '427', '433',
-    '439', '442', '445', '626', '646',
-    '690'
-]
+# Global variables, which we're later going to refactor into an external config file.
+
+topics = ['321', '336', '341', '347', '350', '362', '363', '367', '375', '378',
+          '393', '397', '400', '408', '414', '422', '426', '427', '433', '439',
+          '442', '445', '626', '646', '690']
+
+config = {
+    'target': {'name': 'core18',
+               'run': 'run.core18.bm25+rm3.topics.core18.txt',
+               'index': 'lucene-index.core18.pos+docvectors+rawdocs'}
+}
+
+working_directory = 'ccrf'
 
-def read_vocab(path):
-    """
 
-    """
+def dump_docvectors():
+    run = config['target']['run']
+    source_name = config['target']['name']
+    index = config['target']['index']
+    docids_file = os.path.join(working_directory, f'docids.{source_name}')
+    logging.info(f'Extracting docvectors for {source_name} with Anserini')
+    cmd = f'cut -f 3 -d " " {run} | sort | uniq > {docids_file}'
```

Comment:
just be careful, f-format string method only works in 3.6+, if you consider Python 2/3 compatible code later, should use `''.format()` instead.

Reply:
Thanks for the head's up. We're assuming Python 3.